### ISSUE TITLE
Reword help message for len_zero

### ISSUE THREAD

**Issue by gyscos** [rust-lang-nursery/rust-clippy#3322]
When calling `.len() == 0` on something with a `.is_empty()` method, a lint is triggered with the following help message:
```
help: using `is_empty` is more concise: `self.buffer.is_empty()`
```

Mentioning concision rather than clarity is a bit misleading; `.len() == 0` and `.is_empty()` have actually the *exact same* number of characters.

I suggest rewording it like this:

```
help: using `is_empty` is clearer and more explicit: `self.buffer.is_empty()`
```

(Run with `clippy 0.0.212 (32b1d1f 2018-10-05)`)

**Comment by mrbuzz** [comment]
Hi I'd like to take this as my first issue.

**Comment by CYBAI** [comment]
As #3326 merged, maybe this can be closed? 😄
